guide/07-working-with-bigdata/03-performing_big_data_analysis.ipynb

```diff
@@ -0,0 +1,956 @@
+{
```

Comment:
<p>ArcGIS GeoAnalytics Server tasks that have the <code>outSR</code> property in their Context parameter will save results in the specified spatial reference. If you are saving the results to the spatiotemporal data store, all results will be projected to World Geographic Coordinate System 1984 after analysis for storage, and the <code>outSR</code> will not be used. Set the spatial reference that results will be analyzed in using the Process Spatial Reference property.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>using the Process Spatial Reference property</p><p> </p>